catalog-rest-service/src/main/resources/ui/src/utils/TimeUtils.ts

```diff
@@ -49,13 +49,13 @@ export const getRelativeDayDifference = (
   const elapsed = current - previous;
 
   if (elapsed < msPerDay / 2) {
```

Comment:
This is probably since the beginning, but would like to change it.

Make this `\6`, as `\2` means upto 12 hours, which does not account as few hours.